image/added/concreate/repos/jboss-rhel-rhscl.repo

```diff
@@ -0,0 +1,6 @@
+[jboss-rhel-rhscl]
+name=Red Hat RHEL7 SCL repo
+baseurl=
```

Comment:
why have any repo file with empty base url? this doesn't make sense for me.

Reply:
These are there simply as placeholders.  I wasn't sure what to put in these to make them work ootb, so i left them blank.

Reply:
As these are in the process of being removed, I'd like to hold off removing it here until the other work is complete.